jaeger-core/src/main/java/com/uber/jaeger/samplers/RemoteControlledSampler.java

```diff
@@ -104,6 +105,11 @@ private void updateSampler() {
 
   private Sampler extractSampler(SamplingStrategyResponse response)
       throws UnknownSamplingStrategyException {
+    if(response.getOperationSampling() != null) {
+      return new PerOperationSampler(maxOperations, response.getOperationSampling());
```

Comment:
it looks like we're recreating the whole PerOperationSampler every minute which is not what we want to do. We want to call the Update function instead if the PerOperationSampler already exists.